packages/core/src/schema/columns.ts

```diff
@@ -8,6 +9,43 @@ import type {
   ScalarColumn,
 } from "./common.js";
 
+type DefaultTo<column extends BuilderScalarColumn> = (
+  val: DefaultValue<column[" scalar"]>,
+) => BuilderScalarColumn<
+  column[" scalar"],
+  column[" optional"],
+  column[" list"]
+>;
+
+type DefaultToReturnType<T extends ScalarColumn> = ReturnType<DefaultTo<T>>;
+
+const defaultTo =
+  <column extends BuilderScalarColumn>(col: column): DefaultTo<column> =>
+  (val: DefaultValue<column[" scalar"]>) => {
+    const scalar = col[" scalar"];
+    let v = val;
+    if (scalar === "hex") {
+      let value = val as string;
+      if (value?.startsWith("0x")) {
+        value = value.slice(2);
```

Comment:
What is this for?

Reply:
This is to chop off the 0x from an address or other byte data to be used as default. Otherwise the 0x is stored as hex data, which prefixes the bytes. Here is the test, without that slice: ```

  Array [
    Object {
      "age": 5,
      "bigAge": 5n,
      "bowl": Object {
        "data": Array [
          48,
+         120,
+         48,
          48,
          48,
          48,
          48,
```
as you can see, 0x (48 and 120 respectively) prefix the series of 0s

I now realize, what this really means is that my 0's are being inserted as ascii and not as bytes. I will fix that